main/main.ino

```diff
@@ -453,14 +453,18 @@ void pubMQTT(String topic, unsigned long payload) {
   client.publish((char*)topic.c_str(), val);
 }
 
-void logJson(JsonObject& jsondata) {
+void logJson(String comment, JsonObject& jsondata) {
```

Comment:
Could you use a char * instead? it would avoid the c_.str() below

Reply:
ok!